index.html

```diff
@@ -63,6 +59,169 @@ <h2>I'am a Software Developer</h2>
                 <i class="fa-brands fa-square-twitter"></i>
         </li>
     </ul>
+</div>
+                       <!--My works-->
+<section class="mainmyworks">
+        <div class="worksheading">
+             <div>
+               <h2 class="headtwo">My Recent works</h2>
+               </div>
+         <div>
+                   <hr class="liner">
+               </div>
+        </div>
+        <div class="workitems">
+                    <div class="projectno1">
+                        <div class="photo">
+                        <img src="Images/Img Placeholder.png">
+                        </div>
+                    <div class="page">
+                    <h4 class="headfour">Multi-Post Stories</h4>
+                       <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    </div>     
+                <ul class="pagelinks">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li></div>
+                </ul>
+                <button type="button" class="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="project project2">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+    
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
```

Comment:
- [ ] **[OPTIONAL]** Kindly adjust the size of your cards (and the buttons' size), correct size `h: 386px w: 327px`, to look similar to the figma template. 👇🏽 

| **Figma design** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192119467-c8dc67ac-f430-4bc9-9506-e10820794778.png) | ![image](https://user-images.githubusercontent.com/58642949/192119485-774e4fd3-83c8-4589-a4d8-06a87c5ea74c.png) |

Reply:
[OPTIONAL] I suggest just as the previous reviewer that you increase the height of the cards . This is so that your card design matches the figma design as much as possible

<img width="364" alt="Screenshot 2022-09-26 at 13 16 59" src="https://user-images.githubusercontent.com/48546992/192252829-0cafbac3-758b-43a2-90dd-2bb24211e972.png">

Reply:
Hello @Nyame-Wolf Thanks for your review. I am not sure that I understand what you mean by height, should I increase the background image height or the whole card height?